src/account/utils.py

```diff
@@ -73,7 +73,7 @@ def provider_details(backend=None):
             'backend': 'mit-oidc',
             'name': 'MIT OpenID Connect',
             'org_name': 'MIT',
-            'icon': 'icon-mit',
+            'icon': 'mit-icon-logo',
```

Comment:
The log no longer appears on my local instance. Is it visible in yours?
http://datahub-local.mit.edu/account/add_login